wiki/docs/building.md

```diff
@@ -0,0 +1,26 @@
+Building
+---
+
+With toolchain installed you are ready to compile Mimiker. Before you run
+`make` in project's root directory, please have a look at `config.mk` file.
+
+Of several configurable options, the following are most important to you:
```

Comment:
"the following are most important to you" -- this seems strange to me (how can you know what is important to me?), maybe "the following are the most interesting"? Or maybe "There are several configurable options, including:"